src/include/attribute.h

```diff
@@ -253,6 +253,9 @@ typedef struct ecl_attribute_def ecl_attribute_def;
 #define ATR_VFLAG_HOOK		0x40	/* value set by a hook script   */
 #define ATR_VFLAG_IN_EXECVNODE_FLAG	0x80	/* resource key value pair was found in execvnode */
 
+#define VALUE_DIRTY (ATR_VFLAG_MODIFY | ATR_VFLAG_MODCACHE)
+#define VALUE_SET (ATR_VFLAG_SET | VALUE_DIRTY)
```

Comment:
I'm actually in the process of doing a set/mod/modcache flag in an existing PR of mine.  We should coordinate so we don't do the same thing in different ways.  I think I like my name better because it is more explicit of what values are being set.  Mine is ATR_SET_MOD_MCACHE.  It's a little longer, but is explicit.  We can make the other one just ATR_MOD_MCACHE.

Reply:
sure i can change my names to those